src/DeprecatedTest/UnitTests/constraints.jl

```diff
@@ -520,7 +504,7 @@ function solve_start_soc(model::MOI.ModelLike, config::Config{T}) where {T}
     end
     MOI.empty!(model)
     x = MOI.add_variable(model)
-    fx = MOI.SingleVariable(x)
+    fx = x
```

Comment:
Get rid of `fx`

Reply:
I don't care about these. They're deprecated.